src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -16,14 +16,11 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
```

Comment:
Should we convert this to TS?

Reply:
yeah it shouldn't be a lot of work, but I've been treating it as important technical debt, but non-blocking.